src/main/java/seedu/address/model/transaction/Loan.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.transaction;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents a loan, money which the user lends.
+ */
+public class Loan extends Transaction {
+    /**
+     * Constructs a Loans object.
+     */
+    public Loan(Description description, Amount amount, LocalDate date) {
+        super(description, amount, date, new Tag("Loan"));
+    }
+
+    /**
+     * Checks if both Loans are the same.
+     */
+    @Override
+    public boolean equals(Object other) {
```

Comment:
I feel like equals, hashcode and toString don't need cus Transaction class already have them.

Reply:
But, do we need to differentiate them from other transactions?